modules/library/main/src/main/java/org/geotools/geometry/jts/CompoundCurve.java

```diff
@@ -187,11 +187,15 @@ public String getGeometryType() {
         return "CompoundCurve";
     }
 
-    protected Geometry reverseInternal() {
+    public CompoundCurve reverse() {
+        return (CompoundCurve) super.reverse();
+    }
+
+    public CompoundCurve reverseInternal() {
```

Comment:
Do we want to raise the access here? Internal feels like it shouldn't be public.

Reply:
It should not be, mistake made upstream in JTS (sorry)